### PR TITLE
bs1770gain: update to 0.5.0.beta.5.

### PR DESCRIPTION
**Pull request by autrimpo**
The 0.5 beta branch fixes issues with ffmpeg4.

### CODE REVIEW

File: srcpkgs/bs1770gain/template [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-version=0.4.12
-revision=2
+version=0.5.0.beta.5
+revision=1
+_real_version=0.5.0-beta-5
 build_style=gnu-configure
 makedepends="ffmpeg-devel sox-devel"
 short_desc="A loudness scanner compliant with ITU-R BS.1770"
 maintainer="Michal Koutenský <koutak.m@gmail.com>"
 license="GPL-2"
```
Comment (cr6git) [inline srcpkgs/bs1770gain/template]: `GPL-2.0-or-later`

File: srcpkgs/bs1770gain/template [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 homepage="http://bs1770gain.sourceforge.net"
-distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
-checksum=cafc5440cf4940939c675e98c8dbeb839f4965d60f74270a37d4ee70559b3a59
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${_real_version}.tar.gz"
+checksum=27babaaf257b7d5c8890d82c30594103281c64b85efa231989c66afa09ae804d
+wrksrc="${pkgname}-${_real_version}"
```
Comment (cr6git) [inline srcpkgs/bs1770gain/template]: `wrksrc=` right under `revision=`

Comment (autrimpo) [reply srcpkgs/bs1770gain/template]: pushed a new commit with both changes

**Comment by cr6git** [comment]
https://travis-ci.org/void-linux/void-packages/jobs/419381135#L850

Ah, sorry.
put `wrksrc=` right under `_real_version=`

**Comment by autrimpo** [comment]
No problem, done.

**Pull request merged**